### PR TITLE
[Behat] IBX-1193 added log out step in behat

### PR DESCRIPTION
**Pull request by tomaszszopinski**
| Question      | Answer
| ------------- | ---
| Tickets       |  https://issues.ibexa.co/browse/IBX-1193
| Bug fix?      | no
| New feature?  | no (new tests)
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Related PR: https://github.com/ezsystems/ezplatform-workflow/pull/244

This PR adds a simple step for Behat - logging out of the backoffice.

### CODE REVIEW

**Comment by tomaszszopinski** [comment]
Successful Travis build - https://app.travis-ci.com/github/ezsystems/ezplatform-page-builder/jobs/549797073

File: src/lib/Behat/BrowserContext/NavigationContext.php [hunk 1]
```diff
@@ -135,6 +135,14 @@ public function iGoToUserNotifications()
         $this->upperMenu->chooseFromUserDropdown('View Notifications');
     }
 
+    /**
+     * @Given I log out of backoffice
```
Comment (mnocon) [inline src/lib/Behat/BrowserContext/NavigationContext.php]: Looks like our doc uses "back office": https://doc.ibexa.co/en/latest/extending/extending_back_office/#back-office
```suggestion
     * @Given I log out of back office
```

File: src/lib/Behat/BrowserContext/NavigationContext.php [hunk 2]
```diff
@@ -135,6 +135,14 @@ public function iGoToUserNotifications()
...
+     */
+    public function iLogOutOfBackoffice()
```
Comment (mnocon) [inline src/lib/Behat/BrowserContext/NavigationContext.php]:
```suggestion
    public function iLogOutOfBackOffice()
```

**Review by mnocon — APPROVED** [review]
Looks good, just one small nitpick

**Review by bogusez — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Comment by micszo** [comment]
@tomaszszopinski please merge up/across

**Pull request merged**